apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -1030,7 +1030,92 @@ static void itc_wifi_manager_connect_set_mode_n(void)
 	TC_ASSERT_EQ("wifi_manager_deinit", ret, WIFI_MANAGER_SUCCESS);
 	TC_SUCCESS_RESULT();
 }
+/**
+ * @testcase         itc_wifi_manager_get_stats_p
+ * @brief            get net config information
+ * @scenario         getting network config information
+ * @apicovered       wifi_manager_get_stats
+ * @precondition     none
+ * @postcondition    none
+ */
+static void itc_wifi_manager_get_stats_p(void)
+{
+	wifi_manager_result_e nRet = WIFI_MANAGER_INVALID_ARGS;
+	wifi_manager_stats_s stats;
+	nRet = wifi_manager_get_stats(&stats);
+	if (nRet == WIFI_MANAGER_SUCCESS) {
+		printf("=======================================================================\n");
+		printf("CONN    CONNFAIL    DISCONN    RECONN    SCAN    SOFTAP    JOIN    LEFT\n");
+		printf("%-8d%-12d%-11d%-10d", stats.connect, stats.connectfail, stats.disconnect, stats.reconnect);
+		printf("%-8d%-10d%-8d%-8d\n", stats.scan, stats.softap, stats.joined, stats.left);
+		printf("=======================================================================\n");
+	}
+	TC_ASSERT_EQ("wifi_manager_get_stats_p", nRet, WIFI_MANAGER_SUCCESS);
+	TC_SUCCESS_RESULT();
+}
+/**
+ * @testcase         itc_wifi_manager_get_stats_n
+ * @brief            get net config information for NULL ssid
+ * @scenario         getting network config information for NULL ssid
+ * @apicovered       wifi_manager_get_stats
+ * @precondition     none
+ * @postcondition    none
+ */
+static void itc_wifi_manager_get_stats_n(void)
+{
+	wifi_manager_result_e nRet = WIFI_MANAGER_FAIL;
+	nRet = wifi_manager_get_stats(NULL);
+	TC_ASSERT_EQ("wifi_manager_get_stats", nRet, WIFI_MANAGER_INVALID_ARGS);
+	TC_SUCCESS_RESULT();
+}
+/**
+ * @testcase         itc_wifi_manager_get_connected_config_p
+ * @brief            get connected config information
+ * @scenario         To get connected config information.
+ * @apicovered       wifi_manager_get_connected_config
+ * @precondition     none
+ * @postcondition    none
+ */
 
+static void itc_wifi_manager_get_connected_config_p(void)
+{
+	wifi_manager_result_e nRet = WIFI_MANAGER_FAIL;
+	wifi_manager_ap_config_s apconfig;
+	nRet = wifi_manager_get_connected_config(&apconfig);
+	if (nRet == WIFI_MANAGER_SUCCESS) {
+		printf("====================================\n");
+		printf("SSID: %s\n", apconfig.ssid);
+		printf("SECURITY TYPE: %d\n", apconfig.ap_auth_type);
+		printf("CYPTO TYPE: %d\n", apconfig.ap_crypto_type);
```

Comment:
Can we apply check between retrieved values and expected values to verify output?

Reply:
This comment has been  fixed.